### PR TITLE
Rename stylesheets dir and drop some unused files

### PR DESCRIPTION
**Pull request by orangejenny**
@czue

### CODE REVIEW

**Comment by dimagimon** [comment]
Could not review pull request. It may be too large, or contain no reviewable changes.

File: corehq/apps/hqwebapp/templates/home.html [hunk 1]
```diff
@@ -10,8 +10,8 @@
 {% endblock %}
 {% block head %}{{ block.super }}
     <link rel="stylesheet" href="{% static 'hqwebapp/css/mainlogin.css' %}"/>
-    <link rel="stylesheet" href="{% static 'hqwebapp/stylesheets/blueprint/screen.css' %}" media="screen, projection">
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/home.html]: I suspect this file isn't useful anymore - is there a use case for having prelogin disabled? Guessing we aren't keeping it up to date [("over 10 countries worldwide"!).](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/templates/home.html#L34)

File: corehq/apps/hqwebapp/static/hqwebapp/stylesheets/modules.css [hunk 2]
```diff
@@ -1,11 +0,0 @@
-/* UNICEF RapidSMS UI
```
Comment (czue) [inline corehq/apps/hqwebapp/static/hqwebapp/stylesheets/modules.css]: classic.

File: corehq/apps/hqwebapp/templates/home.html (hunk 1 above)
Comment (czue) [inline corehq/apps/hqwebapp/templates/home.html]: yeah I think we can get rid of it. there was a time when prelogin was a static site you had to build separately but now that it's shipped with HQ i think we can delete this.

**Review by czue — APPROVED** [review]
this is great thanks for digging these up!

**Pull request merged**